extension/agenthealth/handler/useragent/useragent.go

```diff
@@ -78,6 +79,15 @@ func (ua *userAgent) SetComponents(otelCfg *otelcol.Config, telegrafCfg *telegra
 		ua.outputs.Add(output.Config.Name)
 	}
 
+	// Adding SELinux status
+	if os.Getenv("SELINUX") == "true" {
+		ua.inputs.Add("SELinux/Enabled")
```

Comment:
Can you enable http logging and post a screenshot of the UserAgent header to the PR's description